### ISSUE TITLE
How to get fixed source code when using CLIEngine#executeOnText()

### ISSUE THREAD

**Issue by gajus** [eslint/eslint#5003]
```js
import {
    CLIEngine
} from 'eslint';

let cli;

cli = new CLIEngine({
    allowInlineConfig: true,
    baseConfig: false,
    configFile: require.resolve('./eslintrc.json'),
    envs: [],
    extensions: [],
    fix: true,
    globals: [],
    parser: require.resolve('babel-eslint'),
    rulePaths: [],
    useEslintrc: false
});

console.log( cli.executeOnText('var foo;').results );
```

There is nothing in the output that would describe the fixed code. Where do I look for it?

**Comment by eslintbot** [comment]
@gajus Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
`cli.executeOnText('var foo;').results.output` is what will give you the fixed code.

**Comment by gajus** [comment]
@gyandeeps Yes, it appears to be the case. It was not clear because `output` property is missing when there is a syntax error in the file.

**Comment by gyandeeps** [comment]
fyi, output property will be missing if `fix: false` also.

**Comment by mysticatea** [comment]
The output property will be missing if rules make no change. It's convenient.
